proto/dymensionxyz/dymension/sequencer/metadata.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package dymensionxyz.dymension.sequencer;
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/sequencer/types";
+
+import "gogoproto/gogo.proto";
+
+// Metadata defines rollapp/sequencer extra information.
+message SequencerMetadata {
+  // moniker defines a human-readable name for the sequencer.
+  string moniker = 1;
+  // details define other optional details.
+  string details = 2;
+  // bootstrap nodes list
+  repeated string p2p_seeds = 3;
+  // RPCs list
+  repeated string rpcs = 4;
+  // evm RPCs list
+  repeated string evm_rpcs = 5;
+  // REST API URL
+  string rest_api_url = 6;
+  // block explorer URL
+  string explorer_url = 7;
+  // genesis URLs
+  repeated string genesis_urls = 8;
+  // contact details
+  ContactDetails contact_details = 9;
+  // json dump the sequencer can add (limited by size)
+  bytes extra_data = 10;
+  // snapshots of the sequencer
+  repeated SnapshotInfo snapshots = 11;
+  // gas_price defines the value for each gas unit
+  string gas_price = 12 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int"];
```

Comment:
@omritoptix I think this unneeded.
Better way is to have **known** gas price is to put it on-chain

Reply:
this is for UX only I believe